tests/lib/rules/indent.js

```diff
@@ -55,137 +55,65 @@ ruleTester.run("indent", rule, {
     valid: [
         {
             code:
-            "bridge.callHandler(\n" +
```

Comment:
As far as I can tell, all the changes in this file except the first one could be reverted, as the strings were being concatenated for improved readability. The end result is not even a template literal, so this must be a bug with the codemod. 😕